src/apps/btc/btc_sign.c

```diff
@@ -433,7 +447,17 @@ app_btc_sign_error_t app_btc_sign_output(
         next_out->type = BTCSignNextResponse_Type_OUTPUT;
         next_out->index = _index;
     } else {
-        // Done with outputs. Verify total and fee.
+        // Done with outputs. Verify locktime, total and fee.
+        if (_locktime > 0 && _locktime_applies) {
+            // the RBF nsequence bytes only make sense in conjunction with a locktime,
+            // so verify both simultaneously
+            char formatted_locktime[100] = {0};
+            snprintf(
```

Comment:
still a bit confused why you format the locktime here, but rbf inside the function. anyway, check if you can use `workflow_confirm` instead, which would cut down on a lot of code.

Reply:
Is there an easy way to move the confirmation in `workflow_confirm` to the top? Why is it on the bottom in the first place actually?

Reply:
It feels awkward to accept the locktime at the bottom and to the rest of the transaction workflow at the top.

Reply:
i would say

```uint8_t slider_position = scrollable || longtouch ? top_slider : bottom_slider;```

can be made an argument instead in `_confirm_create()`, have `confirm_create()` and `confirm_create_scrollable()` provide the right position, and make a new one `confirm_create_top()` or sth that forces it to the top.

Reply:
Mmh, this is getting a bit complicated for what was before a straight forward change.

Reply:
I'll leave it at the bottom for now, can always improve it later.